### PR TITLE
[#109] Add advisory for sorting arities

### LINKED ISSUE #109: Proposal: Sort arities from fewest to most
so

```Clojure
(fn ([x] ...) ([x y] ...) ([x y z & more] ...))
```

Since the common case of multiple arity functions is that you have several partial application arities, one "full" arity and then a varargs case which is a foldr over the varargs. This is also in keeping with the style clojure.core uses for such functions.

### PR DESCRIPTION
**Pull request by arrdem**
Fixes #109

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -259,6 +259,38 @@ when there are no arguments on the same line as the function name.
         (foo x 1))
       ([x y]
         (+ x y)))
+* <a name="multiple-arity-order"></a> Sort the arities of a function
```
Comment (bbatsov) [inline README.md]: This doesn't look right. The new bullet point seems to be within the code block of the previous bullet point.

**Comment by arrdem** [comment]
Fixed.

**Comment by bbatsov** [comment]
Looks good. Just squash those 2 commits together.

**Comment by arrdem** [comment]
Donezo.

**Pull request merged**